### PR TITLE
[11.0][FIX] stock_request: add expected_date implicit on tests to avoid errors

### PR DESCRIPTION
**Pull request by HviorForgeFlow**
It should avoid intermittent errors when running stock_request test on CI.

@lreficent @jbeficent @max3903

### CODE REVIEW

**Review by max3903 — APPROVED** [review]

**Review by bodedra — APPROVED** [review]
Code review. LGTM :+1:

File: stock_request/tests/test_stock_request.py [hunk 1]
```diff
@@ -157,17 +157,20 @@ def test_defaults_order(self):
             order.location_id, self.warehouse.lot_stock_id)
 
     def test_onchanges_order(self):
+        expected_date = fields.Date.today()
```
Comment (bodedra) [inline stock_request/tests/test_stock_request.py]:
```suggestion
        expected_date = fields.Datetime.now()
```

File: stock_request/tests/test_stock_request.py [hunk 2]
```diff
@@ -862,17 +865,20 @@ def test_allow_virtual_location(self):
         self.assertTrue(order.allow_virtual_location)
 
     def test_onchange_wh_no_effect_from_order(self):
+        expected_date = fields.Date.today()
```
Comment (bodedra) [inline stock_request/tests/test_stock_request.py]:
```suggestion
        expected_date = fields.Datetime.now()
```

**Review by bodedra — CHANGES_REQUESTED** [review]

**Review by etobella — APPROVED** [review]

**Review by rousseldenis — APPROVED** [review]
Code review. Makes sense apart @bodedra remark.

File: stock_request/tests/test_stock_request.py (hunk 1 above)
Comment (HviorForgeFlow) [reply stock_request/tests/test_stock_request.py]: There are more tests that relay on today date. As we only need to ensure that date expected is the same in SRO and SR it's fine.

File: stock_request/tests/test_stock_request.py (hunk 2 above)
Comment (HviorForgeFlow) [reply stock_request/tests/test_stock_request.py]: Same

File: stock_request/tests/test_stock_request.py (hunk 1 above)
Comment (bodedra) [reply stock_request/tests/test_stock_request.py]: @hveficent I am agree with your point. But `expected_date` field type is `datetime`.

**Comment by HviorForgeFlow** [comment]
@bodedra done

**Review by bodedra — APPROVED** [review]

**Pull request merged**